setup.cfg

```diff
@@ -1,18 +1,9 @@
-[aliases]
-test = pytest
-
 [flake8]
 max-line-length = 110
 max-doc-length = 79
 ignore = E133, E226, E228, N802, N803, N806, N812, N813, N815, N816, W503
 exclude =
-  sal_interfaces/*,
-  schema/*,
-  scripts/*,
-  tests/.tests,
+  doc,
+  doc/conf.py,
```

Comment:
I gently suggest making doc/conf.py pass flake8. It makes more a much more pleasant experience when editing the file in some editors. It does require some # noqa entries. Here is an example from ts_utils:
```
from documenteer.conf.pipelinespkg import *  # type: ignore # noqa
import lsst.ts.utils  # noqa

project = "ts_utils"
html_theme_options["logotext"] = project  # type: ignore # noqa
html_title = project
html_short_title = project
# Avoid warning: Could not find tag file _doxygen/doxygen.tag
doxylink = {}  # type: ignore
```